src/main/java/com/csdemo/coffeestore/service/ItemsService.java

```diff
@@ -13,45 +13,34 @@
 import java.util.List;
 
 @Service
-public
-class ItemsService{
-	
-	@Autowired
-	private ItemsRepository itemsRepo;
-	
-	
-	public List < ItemsResponse > getItemsList () {
-		return itemsRepo.findAllItems ( );
-	}
-	
-	public void addItems ( List < ItemsRequest > itemRequests ) {
-		for ( ItemsRequest itemRequest : itemRequests ) {
-			Items item=new Items ( );
-			BeanUtils.copyProperties ( itemRequest , item );
-			Items repoItem=itemsRepo.findByname ( item.getName ( ) );
-			if ( repoItem != null ) {
-				repoItem.setQuantity ( repoItem.getQuantity ( )+item.getQuantity ( ) );
-				repoItem.setPrice ( item.getPrice ( ) );
-				itemsRepo.save ( repoItem );
-			}
-			else {
-				itemsRepo.save ( item );
-			}
-			
-		}
-		
-		
-	}
-	
-	public ResponseEntity deleteItemByName ( String itemName ) {
-		Items repoItem=itemsRepo.findByname ( itemName );
-		if ( repoItem != null ) {
-			itemsRepo.delete ( repoItem );
-			return ResponseEntity.ok ( ).build ( );
-		}
-		else
-			throw new ResourceNotFoundException ( "Item "+itemName+"Not available in the menu" );
-		
-	}
-}
+public class ItemsService {
+
+  @Autowired private ItemsRepository itemsRepo;
+
+  public List<ItemsResponse> getItemsList() {
+    return itemsRepo.findAllItems();
+  }
 
+  public void addItems(List<ItemsRequest> itemRequests) {
+    for (ItemsRequest itemRequest : itemRequests) {
+      Items item = new Items();
+      BeanUtils.copyProperties(itemRequest, item);
+      Items repoItem = itemsRepo.findByname(item.getName());
+      if (repoItem != null) {
```

Comment:
in this if else, save part can be common, since u just have to take care of setting appropriate values in the entity.
so instead of saving entity in if .. else.., setting the entity values are much better...no code duplication or no code redundancy